playwright.config.ts

```diff
@@ -7,7 +7,7 @@ import { devices } from "@playwright/test";
  */
 // require('dotenv').config();
 
-if (process.env.WALLET_CREATE_URL === "https://getalby.com/api/users") {
+if (process.env.ALBY_API_URL === "https://getalby.com/api") {
```

Comment:
Just use the BASE_URL (domain) for the value, append paths in the extension.

Reply:
We cannot do this right now due to inconsistency of app location between getalby.com and app.regtest.getalby.com